File: web3rpc/rpc-specs/code-samples/python/governance/getParams.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockNumber = 89

w3 = Web3(Web3.HTTPProvider(host))
governance_response = w3.governance.get_params(blockNumber)

print(governance_response)

File: web3rpc/rpc-specs/code-samples/javascript/governance/getParams.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockNumber = 89;

    provider.governance.getParams(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()